sys/mips/mips/mpboot.S

```diff
@@ -59,10 +65,38 @@ GLOBAL(mpentry)
 
 	mtc0	zero, MIPS_COP_0_CAUSE	/* clear soft interrupts */
 
+	/* Turn on all coprocessors */
+	mfc0	t2, MIPS_COP_0_STATUS
+#if defined(CPU_BERI)
+#  if defined(CPU_CHERI)
```

Comment:
This looks wrong. Why are we not enabling CP1 on Malta?

Reply:
Good question.  I just followed along with malta_mp.c (see the removed block).